ports/esp32/machine_sdcard.c

```diff
@@ -406,9 +410,20 @@ static mp_obj_t machine_sdcard_make_new(const mp_obj_type_t *type, size_t n_args
             }
         }
         #endif
-
-        DEBUG_printf("  Calling init_slot()");
-        check_esp_err(sdmmc_host_init_slot(self->host.slot, &slot_config));
+        #if MP_SD_LDO
```

Comment:
This macro will need a clearer name, something like `#define MICROPY_BOARD_SDCARD_LDO_CH 4` and then can test `#ifdef MICROPY_BOARD_LDO_CH`.

However as this is only needed for the FireBeetle then suggest moving it to a second PR.